src/app/profile/[id]/page.tsx

```diff
@@ -1,63 +1,21 @@
 'use client';
 
-import { Flex, Box, useTheme, Text } from '@chakra-ui/react';
-import { usePathname } from 'next/navigation';
-import Link from 'next/link';
-import LinkAvatar from '@/ui/LinkAvatar';
+import useUserProfileQuery from '@/queries/user/useUserProfileQuery';
+import { APIUser } from '@/types/user';
+import ProfileInfo from '@/ui/ProfileInfo';
 
-const User = {
-  nickName: '다독이',
-  profileImage: 'https://avatars.githubusercontent.com/u/48359052?v=4',
-  email: 'minjongbaek@gmail.com',
-  job: 'IT/프론트엔드 개발자',
-};
-
-const MyPage = () => {
-  const { nickName, profileImage, email, job } = User;
+interface UserProfilePageProps {
+  params: { id: APIUser['userId'] };
+}
 
-  const theme = useTheme();
+const UserProfilePage = ({ params }: UserProfilePageProps) => {
```

Comment:
지난번에 params.id 구조분해할당하는 방법 물어보신 것 같아서 남겨요!!

```suggestion
const UserProfilePage = ({ params: { id } }: UserProfilePageProps) => {
```